server/src/routes/auth/signUp/signUp.js

```diff
@@ -9,26 +11,32 @@ import User from '../../../models';
 const signUp = async (req, res) => {
   const { username, email, password } = req.body;
 
+  //validation
+  const { error } = signupValidation(req.body);
+  if (error) return res.status(StatusCodes.BAD_REQUEST).send(error.details[0].message);
```

Comment:
Let's have common shape of error response, as I said on our call it will be easier to handle errors on the frontend side if all error come as object. Below we used property `message` to give feedback so let's stick to that

Reply:
ok, my bad, it's important to stick with one approach as you mentioned earlier.